plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/thrift/TestThriftSparkMetastoreUtil.java

```diff
@@ -0,0 +1,197 @@
+                        .build(), actualNotExists);
+
+        HiveColumnStatistics actual = fromMetastoreColumnStatistics(new FieldSchema("c_long", BIGINT_TYPE_NAME, null), columnStatistics, OptionalLong.of(4));
+        assertEquals(HiveColumnStatistics.builder()
+                .setIntegerStatistics(new IntegerStatistics(OptionalLong.of(1), OptionalLong.of(4)))
+                .setNullsCount(0)
+                .setDistinctValuesCount(4)
+                .build(), actual);
+    }
+
+    @Test
+    public void testSparkDoubleStatsToColumnStatistics()
+    {
+        Map<String, String> columnStatistics = Map.of("spark.sql.statistics.colStats.c_double.min", "0.3",
+                "spark.sql.statistics.colStats.c_double.distinctCount", "10",
+                "spark.sql.statistics.colStats.c_double.maxLen", "4",
+                "spark.sql.statistics.colStats.c_double.avgLen", "4",
+                "spark.sql.statistics.colStats.c_double.nullCount", "1",
+                "spark.sql.statistics.colStats.c_double.version", "2",
+                "spark.sql.statistics.colStats.c_double.max", "3.3");
+        HiveColumnStatistics actualNotExists = fromMetastoreColumnStatistics(new FieldSchema("c_double_not_exists", DOUBLE_TYPE_NAME, null), columnStatistics, OptionalLong.of(10));
+        assertEquals(HiveColumnStatistics.builder()
+                .setDoubleStatistics(new DoubleStatistics(OptionalDouble.empty(), OptionalDouble.empty()))
+                .build(), actualNotExists);
+
+        HiveColumnStatistics actual = fromMetastoreColumnStatistics(new FieldSchema("c_double", DOUBLE_TYPE_NAME, null), columnStatistics, OptionalLong.of(10));
+        assertEquals(HiveColumnStatistics.builder()
+                .setDoubleStatistics(new DoubleStatistics(OptionalDouble.of(0.3), OptionalDouble.of(3.3)))
+                .setNullsCount(1)
+                .setDistinctValuesCount(9)
+                .build(), actual);
+    }
+
+    @Test
+    public void testSparkDecimalStatsToColumnStatistics()
+    {
+        Map<String, String> columnStatistics = Map.of("spark.sql.statistics.colStats.c_decimal.min", "0.3",
+                "spark.sql.statistics.colStats.c_decimal.distinctCount", "10",
+                "spark.sql.statistics.colStats.c_decimal.maxLen", "4",
+                "spark.sql.statistics.colStats.c_decimal.avgLen", "4",
+                "spark.sql.statistics.colStats.c_decimal.nullCount", "1",
+                "spark.sql.statistics.colStats.c_decimal.version", "2",
+                "spark.sql.statistics.colStats.c_decimal.max", "3.3");
+        HiveColumnStatistics actualNotExists = fromMetastoreColumnStatistics(new FieldSchema("c_decimal_not_exists", DECIMAL_TYPE_NAME, null), columnStatistics, OptionalLong.of(10));
+        assertEquals(HiveColumnStatistics.builder()
+                .setDecimalStatistics(new DecimalStatistics(Optional.empty(), Optional.empty()))
+                .build(), actualNotExists);
```

Comment:
```
 assertEquals(
                HiveColumnStatistics.builder()
                     .setDecimalStatistics(new DecimalStatistics(Optional.empty(), Optional.empty()))
                     .build(), 
                actualNotExists);
```

Reply:
Could you add more context to your comment?